lighthouse-core/report/html/report-styles.css

```diff
@@ -853,18 +853,20 @@
 }
 
 .lh-scorescale-range {
+  display: flex;
+  align-items: center;
   margin: 0 12px;
   font-family: var(--monospace-font-family);
   white-space: nowrap;
 }
 
 .lh-scorescale-range::before {
   content: '';
-  width: var(--body-font-size);
-  height: calc(var(--body-font-size) * .50);
+  width: 18px;
```

Comment:
should we var these babies up somewhere?

Reply:
done